### ISSUE TITLE
Figure out engineer SSH access to Lagoon instances.

### ISSUE THREAD

**Issue by ndouglas** [department-of-veterans-affairs/va.gov-cms#13268]

**Comment by olivereri** [comment]
For vagov-dev using AWS creds I've been able to "ssh" into pods using AWS CLI's EKS commands. This may just be a matter of getting proper permissions. Though, we are already admins so this _might_ be a non-issue.

**Comment by ndouglas** [comment]
Yeah, you're right. I did that as part of #6674 when troubleshooting various issues. Maybe I blocked it out? 🤔 I'll review that tomorrow to remind myself how it worked.
